fxapom/fxapom.py

```diff
@@ -54,6 +55,16 @@ def create_account(self):
         self.session.verify_email_code(m["headers"]["x-verify-code"])
         return self
 
+    def delete_account(self):
+        if not self.deleted:
```

Comment:
We get an exception if we try to delete an account that doesn't exist. I have reworked the code to catch this exception.